### PR TITLE
fix: Use embedded images for storybook

### LINKED ISSUE #1265: Happo failures when referencing external images in storybook
We've been seeing Happo diffs when referencing external images. Try using base64 images in the stories to remove external dependencies

### PR DESCRIPTION
**Pull request by ahobson**
# Summary

Try embedded images to fix Happo diffs

## Related Issues or PRs

closes #1265

## How To Test

See if spurious Happo diffs go away

### CODE REVIEW

**Comment by trussworks-infra** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>This PR does not include changes to tests, even though it affects source code.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6f5897cb4dd7c0f297daa9105945ea5507b8b87b
</p>

File: src/components/card/Card.stories.tsx
Comment (brandonlenz) [inline src/components/card/Card.stories.tsx:259]: Not sure if we care, but some of these images are not the original images (noticed it in Happo):
image

File: src/components/card/Card.stories.tsx
Comment (brandonlenz) [inline src/components/card/Card.stories.tsx:23]: What are your thoughts on moving these to a separate file for storybook constants?

**Review by brandonlenz — COMMENTED** [review]
Couple of questions before I approve, but I'm very excited for this to unblock dependabot updates 😄 🚀 🥳

File: src/components/card/Card.stories.tsx
Comment (ahobson) [reply src/components/card/Card.stories.tsx:259]: Yeah, I should have mentioned that! Some of the images seemed to be only used a few times and I didn't see a reason why we couldn't pare down our list.

If I'm wrong, I'm happy to change it

File: src/components/card/Card.stories.tsx
Comment (ahobson) [reply src/components/card/Card.stories.tsx:23]: I looked into that and wondered what the right approach is. We don't want these included in our final build and I think that would mean they should be excluded in `tsconfig.json`. Should we add `**/*.stories.ts`? Or put this in another `stores.tsx` file that doesn't have any components? Other ideas?

Comment (brandonlenz) [reply src/components/card/Card.stories.tsx:23]: Good points. Would it be worth having multiple tsconfig files like [this](https://stackoverflow.com/questions/58461649/how-to-exclude-specific-files-in-typescript-only-for-the-build)?

Our current tsconfig exclusions actually leave us with false positives for import paths for example:
image.

Moving all of those into a `tsconfig.publish.json` or something would resolve that small annoyance as well.

It starts to get funky trying to specify that specific tsconfig file for certain scripts, and that's about where my knowledge drops off.

> should we add `**/*.stories.ts`?

This is the easiest solution, and just requires some diligence with naming convention, but I like it.

Comment (brandonlenz) [reply src/components/card/Card.stories.tsx:23]: Hmmm you know what I didn't notice these are at least all on one line within the storybook file. I just saw the wrapping in the github UI. I'm good with it as is for getting this out the door. Its always something we can improve later.

**Review by brandonlenz — APPROVED** [review]

File: src/components/card/Card.stories.tsx
Comment (ahobson) [reply src/components/card/Card.stories.tsx:23]: Thanks for looking at this. I rebased it on my machine instead of doing that through the GitHub UI and now I need approval again. 😅

**Review by brandonlenz — APPROVED** [review]
@ahobson ✅ 

Looking forward to watching those dependabot PRs fly!

**Pull request merged**
